packages/docusaurus-plugin-client-redirects/package.json

```diff
@@ -32,6 +32,7 @@
     "@docusaurus/types": "3.6.1"
   },
   "peerDependencies": {
+    "@mdx-js/react": "^3.0.0",
```

Comment:
For each of the peers added in this PR, there were previously warnings like:

```
➤ YN0002: │ @docusaurus/plugin-client-redirects@workspace:packages/docusaurus-plugin-client-redirects doesn't provide @mdx-js/react (p31cf2), requested by @docusaurus/core
➤ YN0002: │ @docusaurus/plugin-client-redirects@workspace:packages/docusaurus-plugin-client-redirects [86937] doesn't provide @mdx-js/react (pc71f5), requested by @docusaurus/core
```

I've added additional peer dependency definitions as necessary, and then the dependency itself to the top level package (e.g., `website`).